cs-commons/src/main/java/io/cloudslang/content/utils/BooleanUtilities.java

```diff
@@ -0,0 +1,61 @@
+package io.cloudslang.content.utils;
+
+import io.cloudslang.content.constants.BooleanValues;
+import io.cloudslang.content.constants.ExceptionsValues;
+import org.apache.commons.lang3.BooleanUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Do we put jetbrains jar on classpath for this cool annotations? If this is the case maybe it is a bit to much, not sure about the license and so on